README.md

```diff
@@ -1 +1,237 @@
-# budget-app
\ No newline at end of file
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
```

Comment:
- [ ] In the recorded video, kindly address a piece of code or feature that you liked the most.